ethereum_history_api/circuits/lib/src/misc/fragment.nr

```diff
@@ -24,16 +24,20 @@ impl<MAX_DATA_LEN, T> Fragment<MAX_DATA_LEN, T> {
         Fragment { offset: 0, length, data }
     }
 
-    pub fn from_array<DATA_LEN>(data: [T; DATA_LEN]) -> Fragment<MAX_DATA_LEN, T> {
+    pub fn from_array<DATA_LEN>(data: [T; DATA_LEN]) -> Fragment<DATA_LEN, T> {
```

Comment:
Do we need another generic here? Can't we reuse MAX_DATA_LEN?

Reply:
You're right, we can. Changed it